cmd/ui/src/rendering/programs/edge.curvedArrowHead.ts

```diff
@@ -150,9 +151,16 @@ export default class CurvedEdgeArrowHeadProgram extends AbstractEdgeProgram {
             return;
         }
 
+        let start, end: Coordinates;
+        if (data.direction === EdgeDirection.BACKWARDS) {
+            end = { x: sourceData.x, y: sourceData.y };
+            start = { x: targetData.x, y: targetData.y };
```

Comment:
Small nit but by having both variables defined in the same order, I think it is a tad easier to see the difference here as well. 
```suggestion
            start = { x: targetData.x, y: targetData.y };
            end = { x: sourceData.x, y: sourceData.y };
```

Reply:
✔️